### PR TITLE
Add Polish translation

### PR DESCRIPTION
**Pull request by andriej**
Add Polish translation

### CODE REVIEW

**Comment by andriej** [comment]
@bieniu please take a look :-)

**Review by bieniu — APPROVED** [review]
LGTM 👍

**Review by denysdovhan — APPROVED** [review]

File: src/translations/en.json [hunk 1]
```diff
@@ -1,38 +0,0 @@
-{
```
Comment (denysdovhan) [inline src/translations/en.json:1]: Please, bring back the English translation. It's the main one.

**Review by denysdovhan — CHANGES_REQUESTED** [review]
Do not delete English translation. I believe that's by mistake.

File: src/translations/en.json (hunk 1 above)
Comment (andriej) [reply src/translations/en.json]: Totally by mistake, didn't notice. Sorry!

**Review by andriej — COMMENTED** [review]
en.json is back in PR

**Review by denysdovhan — APPROVED** [review]

**Comment by denysdovhan** [comment]
Thanks for contributing!

**Pull request merged**